packages/assemble-lite/helper/helpers-helper.js

```diff
@@ -0,0 +1,28 @@
+
+
+const { resolve } = require("path");
+const handlebarsHelpers = require("handlebars-helpers");
+
+const { getPaths } = require("./io-helper");
+
+
+const loadHelpers = async (helpers, hbsInstance) => {
+
+  hbsInstance.registerHelper(handlebarsHelpers());
+
+  const helperPaths = await getPaths(helpers);
+  helperPaths.forEach(path => {
+    try {
+      const helperFkt = require(resolve(path));
```

Comment:
what does helperFkt mean ? If you mean function here then better use helperFn

Reply:
fixed